public/assets/scss/_index.scss

```diff
@@ -291,6 +291,11 @@ h5 .btn-link {
     .faq {
         padding-left: 3rem;
     }
+    .about-ball-heading {
+        br {
```

Comment:
I believe this can be manipulated via bootstrap's display class. Adding a class like `.d-none .d-lg-block` to the affected <br> should do the trick. https://getbootstrap.com/docs/4.0/utilities/display/

Reply:
doesn't work

Reply:
I am sure you didn't go through the link or try to know how this works. This works with  
```html
<h2 class="about-ball-heading">Game-changing service <br class="d-none d-xl-block"> that makes owning <br class="d-none d-xl-block"> your home easier</h2>
```